extra_requirements/nasbench_1shot1.json

```diff
@@ -0,0 +1,3 @@
+{
+  "nasbench_1shot1": ["tensorflow==1.15.0","matplotlib","seaborn", "networkx", "tqdm"]
```

Comment:
Sorry, I realize this only now: But why do we need matplotlib and seaborn? We don't do any plotting in the container, do we?

Reply:
Jep. That's right. But Arber's code needs it and there is no setup.py in his code. So, we need to install the necessary requirements here.